langchaingo/compose.openai.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+  app:
+    environment:
+      - OPENAI_MODEL_NAME=gpt-4.1
+    secrets:
+      - openai-api-key
+
+secrets:
+  openai-api-key:
+    file: secret.openai-api-key
```

Comment:
issue: add a `.gitignore` in this directory to prevent accidentally comitting this file, please